### ISSUE TITLE
Custom module causes error 500 when saved but record still saves

### ISSUE THREAD

**Issue by PercyP** [YetiForceCompany/YetiForceCRM#4579]
Hi,

I have a custom module which triggers an error 500 with white screen every time a record is saved. If I refresh the page whilst the error is on the page, and then go view the record, the new details are saved. If I edit a field value without going in to the full edit page it also saves without causing the error. 

When I look at the url where the error occurs (after clicking on Save) it has /index.php at the end and the 'Secure' is not showing in the url bar. All other pages have the word Secure in the url.

Is there an obvious cause/fix for this? 

Kind regards

**Comment by mariuszkrzaczkowski** [comment]
Check logs

**Comment by PercyP** [comment]
@mariuszkrzaczkowski,

thanks for responding.
I have decided to rebuild the module from scratch as something is obviously wrong with the module itself. It would seem to be quicker to rebuild than go through all the troubleshooting. 

Kind regards

**Comment by mariuszkrzaczkowski** [comment]
If you have any problems always check the logs there for information that does not work

**Comment by PercyP** [comment]
Hi,

just in case others have the same problem, I think I know what caused this error (as just done it again). I had deleted the field 'title' (which is the text field we have to use when creating a new module. I couldn't delete it from the Edit fields bit as it was mndatory and wouldn't allow me to make it none mandatory, so I removed it from the database and made another field mandatory instead. As a result, it was still trying to save to the missing field and producing the error 500. In hindsight I should have just made it none mandatory in the database and then gone back to edit fields and disabled it. When I re-added the field 'title' back in the database the module saved correctly. 

I guess the moral of the story is not to mess with the database!
